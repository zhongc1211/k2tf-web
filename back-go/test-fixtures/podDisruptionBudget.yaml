apiVersion: policy/v1beta1
kind: PodDisruptionBudget
metadata:
  name: test
spec:
  minAvailable: 75%
  selector:
    matchLabels:
      app: zookeeper