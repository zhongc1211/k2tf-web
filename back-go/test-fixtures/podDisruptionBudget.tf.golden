resource "kubernetes_pod_disruption_budget" "test" {
  metadata {
    name = "test"
  }
  spec {
    min_available = "75%"
    selector {
      match_labels = { app = "zookeeper" }
    }
  }
}
